website/siteConfig.js

```diff
@@ -33,7 +33,7 @@ const siteConfig = {
     organizationName: "textlint", // or set an env variable ORGANIZATION_NAME
     projectName: "textlint",
     headerLinks: [
-        { doc: "README", label: "Docs" },
+        { doc: "index", label: "Docs" },
```

Comment:
```diff
- index
+ getting-started
```

Reply:
Agreed. This is what I meant at https://github.com/textlint/textlint/pull/468#discussion_r161517368.